code/game/gamemodes/game_mode.dm

```diff
@@ -637,6 +637,16 @@
 		if(SSshuttle.emergency.is_hijacked())
 			SSticker.news_report = SHUTTLE_HIJACK
 
+/**
+  * Given a list of minds, returns TRUE if they completed all their objectives, FALSE otherwise
+  */
+/datum/game_mode/proc/didAntagsWin(list/antags)
```

Comment:
Can you add a second argument being an antagonist type and filtering the antag datums by that? So that if an admin gives the Ai a custom antag status of bringer or doom or whatever it doesnt interfere with the win state